pkgs/tools/misc/sensible-utils/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchgit, makeWrapper, bash }:
+
+let
+  pname = "sensible-utils";
+  version = "0.0.18";
+in
+stdenv.mkDerivation rec {
+  inherit pname version;
+
+  # https://salsa.debian.org/debian/sensible-utils
+  src = fetchgit {
+    url = "https://salsa.debian.org/debian/sensible-utils.git";
+    rev = "debian/${version}";
+    sha256 = "sha256-fZJKPnEkPfo/3luUcHzAmGB2k1nkA4ATEQMSz0aN0YY=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+    # We don't want to use configure!
+  '';
```

Comment:
```suggestion
  dontConfigure = true;
```

Reply:
Ah, there is an option for this! Thank you!